charts/timescaledb-single/templates/statefulset-timescaledb.yaml

```diff
@@ -37,6 +37,83 @@ spec:
         # The postgres user inside the TimescaleDB image has uid=1000.
         # This configuration ensures the permissions of the mounts are suitable
         fsGroup: 1000
+{{- if .Values.timescaledbTune.enabled }}
+      initContainers:
+      - name: tstune
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

Comment:
This is the same image as we use for PostgreSQL, so it's going to be cheap to use that image.
Going forward however, we could use an image containing *only* timescaledb-tune?